### ISSUE TITLE
[New Port Request] gecode

### ISSUE THREAD

**Issue by ww-rm** [microsoft/vcpkg#35979]
### Library name

gecode

### Library description

Gecode - Generic Constraint Development Environment

### Source repository URL

https://github.com/Gecode/gecode

**Comment by waywardmonkeys** [comment]
I've done some work in this direction before and I need to submit a couple of PRs to Gecode and get them merged ... Happy to finish this up over the next week or so (have a lot of travel in the coming days).

**Comment by waywardmonkeys** [comment]
There is some activity upstream to enable this.

**Comment by ww-rm** [comment]
Hi, thank you for your continuous efforts on this issue. However, due to my personal progress, I have to try using 'vcpkg install gecode' on the Windows system myself.

I emulated the writing style of other port files, added the gecode repository for version 6.3.0, and attempted to install it using 'vcpkg install gecode'. But there was an error; the error message indicated that cmake could not find the library files during the installation. I checked the output directory and found that only *.dll files were generated, with no *.lib files present. Does this mean that installing gecode on Windows via vcpkg requires some additional configuration?

Moreover, I found a similar problem in #14091 , https://github.com/microsoft/vcpkg/pull/14091#issuecomment-712642831, and I am looking forward to your reply in your busy schedule!
